### PR TITLE
[MODULAR] New HoP alternate job titles

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request
Adds a couple Human Resources style alternate job titles for the HoP.

## How This Contributes To The Skyrat Roleplay Experience
Adds more flavour, the HoP is basically HR for the station.

## Changelog

:cl: LT3
add: New alternate job titles for the HoP.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 1]
```diff
@@ -81,7 +81,7 @@
 	alt_titles = list("Geneticist", "Mutation Researcher")
 
 /datum/job/head_of_personnel
-	alt_titles = list("Head of Personnel", "Executive Officer", "Employment Officer", "Crew Supervisor")
+	alt_titles = list("Head of Personnel", "Executive Officer", "Employment Officer", "Crew Supervisor", "Enterprise Resource Planner", "Chief Resources Officer")
```
Comment (Nerev4r) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: Let's not add 'ERP HOP' and pretend we did.  This doesn't even fit anything else here.

Comment (lessthnthree) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: I didn't want to call it Human, because we have a bunch of species.

Comment (Nerev4r) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: It's a joke name, sir.

Comment (Iamgoofball) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: enterprise resource planner is also better as a QM name, but also lets not add ERP HOP

**Comment by YakumoChen** [comment]
ERP aside "Chief Resources Officer " sounds more like a QM name

Why not make use of the word "Staffing" or something.

Or add something like Civilian Management or Service Coordinator half a dozen other aspects of HoP that nobody uses

**Comment by lessthnthree** [comment]
> ERP aside "Chief Resources Officer " sounds more like a QM name
> 
> Why not make use of the word "Staffing" or something.
> 
> Or add something like Civilian Management or Service Coordinator half a dozen other aspects of HoP that nobody uses

Chief Services Coordinator sounds good.

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 2]
```diff
@@ -81,7 +81,7 @@
 	alt_titles = list("Geneticist", "Mutation Researcher")
 
 /datum/job/head_of_personnel
-	alt_titles = list("Head of Personnel", "Executive Officer", "Employment Officer", "Crew Supervisor")
+	alt_titles = list("Head of Personnel", "Executive Officer", "Employment Officer", "Crew Supervisor", "Enterprise Resource Planner", "Chief Services Coordinator")
```
Comment (Nerev4r) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
	alt_titles = list("Head of Personnel", "Executive Officer", "Employment Officer", "Crew Supervisor", "Chief Service Coordinator")
```

Comment (Iamgoofball) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: ░░░░░░░░░░░░▄▄                     
░░░░░░░░░░░█░░█                  
░░░░░░░░░░░█░░█                  
░░░░░░░░░░█░░░█                  
░░░░░░░░░█░░░░█                  
███████▄▄█░░░░░██████▄ 
▓▓▓▓▓▓█░░░░░░░░░░░░░░█ 
▓▓▓▓▓▓█░░░░ Valve, ░░░░░█ 
 ▓▓▓▓▓▓█░░░░Add This░░░░█  
▓▓▓▓▓▓█░░░░Please!░░░░░█ 
▓▓▓▓▓▓█░░░░░░░░░░░░░░█ 
▓▓▓▓▓▓█████░░░░░░░░░█   
██████▀░░░░▀▀██████▀

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm (hunk 1 above)
Comment (CandleJaxx) [reply modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]: erotic role play head of personnel

**Comment by lessthnthree** [comment]
CRO really is more of a QM title.

**Pull request closed without merging**